index.html

```diff
@@ -1,143 +1,54 @@
-        </div>
-      </div>
-    <% }); %>
-    <hr class="bold" />
-    <div class="row form-inline">
-      <div class="col-2 col-md-1"><label><b>New</b></label></div>
-      <div class="col-4 col-md-3 col-lg-4"><input class="form-control" type="text" id="guest_name_new" value="" placeholder="Name" /></div>
-      <div class="col-4 col-md-3"><input class="form-control" type="text" id="guest_position_new" value="" placeholder="Position" /></div>
-      <div class="col-2 col-md-1"><input class="form-control" type="checkbox" disabled /></div>
-      <div class="col-12 col-md-4 col-lg-3 text-right text-md-left">
-        <div class="mt-3 d-block d-md-none"></div>
-        <input type="button" class="btn btn-success" value="Add player" onclick="add_guest(get_query_value('event_id'))" />
-      </div>
-    </div>
-  </script>
-  <script>
-    var template = _.template($( "#players-template" ).html());
-    $( "#players-template" ).remove();
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <link rel="shortcut icon" href="">
+  <title>Let's volleyball</title>
 
-    $(document).ready(function() {
-      $.ajax({type: "POST",
-        url: apiLink,
-        data: {
-          action: 'event',
-          id: get_query_value('event_id')
-        },
-        success: function(response){
-          var result = JSON.parse(response);
-          if (result.status != 0) {
-            alert(result.message);
-          } else {
-            const parse_format = 'DD.MM.YYYY HH:mm';
-            const display_format = 'dddd, MMM Do YYYY [at] HH:mm';
-            const m_date = moment(result.message.date, parse_format);
-            $('#event_date').text(m_date.format(display_format));
-            $('#event_location').text(result.message.location);
-            // update both text and href attribute
-            $('#payment_link').text(result.message.payment_link);
-            $('#payment_link').attr('href', result.message.payment_link);
+  <!--  css -->
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic|Anonymous+Pro:400,700,400italic,700italic|Merriweather:400,700,300">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="src/css/main.css?v=1.0.2">
 
-            $('.addeventatc .timezone').text('Europe/Amsterdam');
-            $('.addeventatc .title').text('Let\'s volleyball @' + result.message.location + ' on ' + m_date.format(display_format));
-            $('.addeventatc .description').text(result.message.date + ' ' + result.message.location);
-            $('.addeventatc .location').text(result.message.location);
+  <!--  js -->
+  <script src="https://cdn.jsdelivr.net/npm/vue@2.5.16/dist/vue.min.js"></script>
```

Comment:
Would `vue.min.js` replace jquery?

Reply:
I'm still using `$.ajax`, so still need that bit. Gonna work to change it to [fetch](https://developers.google.com/web/updates/2015/03/introduction-to-fetch)